tests/common.go

```diff
@@ -7180,24 +7180,22 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
 		AfterEachTest(contexts, CurrentTestRailTestCaseId, RunIdForSuite)
 	}
 	ginkgoTestDescr := ginkgo.CurrentGinkgoTestDescription()
-	if ginkgoTestDescr.Failed {
-		log.Infof(">>>> FAILED TEST: %s", ginkgoTestDescr.FullTestText)
```

Comment:
Let us retain this log message `log.Infof(">>>> FAILED TEST: %s", ginkgoTestDescr.FullTestText)`

Reply:
retained the failed test comment